adapters/deutsche-bank-adapter/src/main/java/de/adorsys/xs2a/adapter/service/impl/DeutscheBankAccountInformationService.java

```diff
@@ -19,38 +19,33 @@
 import de.adorsys.xs2a.adapter.adapter.BaseAccountInformationService;
 import de.adorsys.xs2a.adapter.http.ContentType;
 import de.adorsys.xs2a.adapter.http.HttpClient;
+import de.adorsys.xs2a.adapter.http.Request.Builder.Interceptor;
 import de.adorsys.xs2a.adapter.service.RequestHeaders;
 
-import java.time.ZonedDateTime;
-import java.time.format.DateTimeFormatter;
 import java.util.Map;
 
 public class DeutscheBankAccountInformationService extends BaseAccountInformationService {
-    private static final String DATE_HEADER = "Date";
 
-    public DeutscheBankAccountInformationService(String baseUri, HttpClient httpClient) {
-        super(baseUri, httpClient);
+    public DeutscheBankAccountInformationService(String baseUri, HttpClient httpClient, Interceptor interceptor) {
+        super(baseUri, httpClient, interceptor);
     }
 
     @Override
     protected Map<String, String> populateGetHeaders(Map<String, String> map) {
         Map<String, String> headers = super.populateGetHeaders(map);
-        headers.put(DATE_HEADER, DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now()));
```

Comment:
I'm not sure, that it is a good approach to split the responsibility of populating headers between the 2 services (this class and the interceptor) whenever it is not necessary. 
I see, that there is a Sonar complains about the duplicates, but in my opinion, we'd better proceed with duplicates rather than decrease the readability (now it is not that obvious, that current adapter requires changing any additional header except the `CONTENT_TYPE` one) and provide a reason for the request to be intercepted one more time before being finally sent.

Reply:
Reverted.